functests/1_performance/performance.go

```diff
@@ -200,13 +200,14 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 				Expect(err).ToNot(HaveOccurred())
 			}
 		})
-		It("[test_id:42400][crit:medium][vendor:cnf-qe@redhat.com][level:acceptance] stalld daemon is running as sched_fifo", func() {
+		FIt("[test_id:42400][crit:medium][vendor:cnf-qe@redhat.com][level:acceptance] stalld daemon is running as sched_fifo", func() {
                          for _, node := range workerRTNodes {
-                                 pid, err :=nodes.ExecCommandOnNode([]string{"pgrep", "-f","stalld"},&node)
+                                 pid, err := nodes.ExecCommandOnNode([]string{"pgrep", "-f", "stalld"}, &node)
                                  Expect(err).ToNot(HaveOccurred())
-                                 sched_tasks,_:=nodes.ExecCommandOnNode([]string{"chrt","-ap",pid},&node)
-                                 Expect(strings.Contains(sched_tasks,"scheduling policy: SCHED_FIFO") && strings.Contains(sched_tasks,"scheduling priority: 10")).To(Equal(true))
-                         }
+                                 sched_tasks,_:=nodes.ExecCommandOnNode([]string{"chrt", "-ap", pid}, &node)
+				 Expect(sched_tasks).To(ContainSubstring("scheduling policy: SCHED_FIFO"))
+				 Expect(sched_tasks).To(ContainSubstring("scheduling priority: 10"))
```

Comment:
still weird spacing, and the CI lane is probably not going to like it

Reply:
Thanks, reformatted the code , please let me know if I missed anything.